W8 - Portfolio-Maria

Maria-Manuela opened the pull request:
## Netlify link
https://bmmportfolio.netlify.app/

Maria-Manuela committed:
components created

Maria-Manuela committed:
icons

Maria-Manuela committed:
intro section

Maria-Manuela committed:
setting the base for colors, fonts. Intro-done

Maria-Manuela committed:
tech sections added, picture and icons.

Maria-Manuela committed:
all changes

Maria-Manuela committed:
error again, commit for restart

Maria-Manuela committed:
section projects- in progres

Maria-Manuela committed:
API repo fetched. Label added.Styling in progres...

Maria-Manuela committed:
section feature projects-done

Maria-Manuela committed:
my words section-done. kinda

Maria-Manuela committed:
skills section

Maria-Manuela committed:
all the sections done

Maria-Manuela committed:
last changes, project still under progress

Maria-Manuela committed:
small changes

HIPPIEKICK reviewed (changes requested):
Hi! Please make sure you're following the design and that your project is responsive (have a look at 320px width). Here are some questions to start from:

- Are all font-sizes according to the design?
- Are all font-families according to the design?
- Are all sizes according to the design?
- Are all distances (margin, padding) according to the design? Are the sections aligned where they should be?
- Does it look good on all screen widths from 320px to 1600px?
- Are the line-heights according to the design?
- Are the texts center/left aligned according to the design?

Maria-Manuela committed:
styling adjusted for intro and tech section. Projects styling in prongress...

Maria-Manuela committed:
responsiveness changes

Maria-Manuela committed:
update

Maria-Manuela committed:
Update README.md

Maria-Manuela committed:
all projects added

Maria-Manuela committed:
Merge branch 'main' of https://github.com/Maria-Manuela/project-portfolio

HIPPIEKICK commented:
Keep up the good work, you're getting there 👏 Let us know if you need any assistance.

Maria-Manuela committed:
adjusting responsiveness

Maria-Manuela committed:
progress..

Maria-Manuela committed:
project images adjusted

Maria-Manuela committed:
adjustements

NellaMori reviewed (changes requested):
Hi MM, I can see you applied some changes, but the font sizes are still not exactly the same ones of the design. Please copy all the css rules in terms of sizes, margins and paddings, this is part of the exercise and assignment. Follow the design layout by the pixel :)

Maria-Manuela committed:
Improve portfolio responsiveness and project display across breakpoints.

Unify section padding with CSS variables, refactor GitHub project fetching to follow projects.json order with a Show More toggle, and fix project/article card image layout to match Figma mobile and tablet specs.

Maria-Manuela committed:
Update ReadMe.md
